Document/MediaManager.php

```diff
@@ -30,11 +30,13 @@ public function save($entity, $andFlush = true)
     {
         // BC compatibility for $context parameter
         if ($andFlush && is_string($andFlush)) {
+            trigger_error('Since version 2.2.7, passing context in argument 2 for '.__METHOD__.'() is deprecated and support for it will be removed in 3.0. Use '.$this->class.'::setContext() instead.', E_USER_DEPRECATED);
```

Comment:
> __METHOD__ already includes the QCN, so your suggestion will output something like

Indeed, you are right.

> Check also this Symfony's deprecation message, it's practically identical.

Well, seems deprecation messages are not the same on all of the Symfony code. BTW, for this, it's was just a suggestion. ;-)